### PR TITLE
Create FUNDING.md

### PR DESCRIPTION
**Pull request by FredKSchott**
## Changes

- Adds a FUNDING.md page to describe how Open Collective funds are collected and distributed.
- The goal here is to bring transparency to the new fundraising/sponsorship process.
- This is still early, so not much to report in this document other than transparency into our plans for distribution. I stop short of committing to regular updates in this document, but the goal here is to come back and update this document around once per quarter.

### CODE REVIEW

File: FUNDING.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+Below is a (non-exhaustive) list of how we plan to distribute raised funds:
+
+- **Swag!** Creating stickers, t-shirts, etc. for sponsors and community members.
+- **Improve documentation.**
+- **Improve translations.**
+- **Improve website.**
+- **User research.**
+- **Sponsoring conferences.**
+- **Sponsoring community memebers to represent Astro at meetups, conferences, etc.**
```
Comment (jasikpark) [inline FUNDING.md:35]: "community meme-bers 😆"
```suggestion
- **Sponsoring community members to represent Astro at meetups, conferences, etc.**
```

**Review by jasikpark — COMMENTED** [review]
exciting!

File: FUNDING.md (hunk 1 above)
Comment (FredKSchott) [reply FUNDING.md]: lol :)

**Comment by FredKSchott** [comment]
Needed for 0.19 release post, so merging assuming no issues!

**Pull request merged**